beacon-chain/rpc/beacon/blocks.go

```diff
@@ -149,10 +164,15 @@ func (bs *Server) ListBlocks(
 		if err != nil {
 			return nil, err
 		}
+		canonical, err := bs.CanonicalFetcher.IsCanonical(ctx, root)
+		if err != nil {
+			return nil, err
+		}
 		containers := []*ethpb.BeaconBlockContainer{
 			{
 				Block:     genBlk,
 				BlockRoot: root[:],
+				Canonical: canonical,
```

Comment:
A genesis block should always be canonical. We can just do

```suggestion
				Canonical: true,
```

Reply:
Yes, that's true. But I am hesitate that RPC server has this knowledge (genesis must be canonical), so I end up asking the `blockchain` about it. What do you think @terencechain?

Reply:
I think hard coding this logic is fine since this is a general concept in eth2. Genesis slot is always canonical and finalized